### PR TITLE
fix and update getHeader & setHeaders functions

### PR DESCRIPTION
**Pull request by lamhieu-vk**

### CODE REVIEW

**Comment by lamhieu-vk** [comment]
- fix getHeader function at [commit](https://github.com/mafintosh/turbo-http/pull/6/commits/dccdec5aaa06913137d46a57e65a365d050d8ee8)
- add setHeaders function to quickly set multi headers, at [commit](https://github.com/mafintosh/turbo-http/pull/6/commits/f9c0386107f95dc7286b93374dcb1c8717d69cf8)

File: index.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ class Request {
   }
 
   getHeader (name) {
-    this.getAllHeaders().get(name)
+    return this._headers && this._headers.get(name) || this.getAllHeaders().get(name)
```
Comment (mafintosh) [inline index.js]: could you make this ternary instead? `this._header ? this._header.get(name) : this.getAllHeaders().get(name)`

File: index.js [hunk 2]
```diff
@@ -65,6 +65,12 @@ class Response {
     else if (CONNECTION.test(name)) this._keepAlive = false
   }
 
+  setHeaders (obj) {
+    Object.keys(obj).map(key => {
+      this.setHeader(key, obj[key])
+    })
```
Comment (mafintosh) [inline index.js]: could we make this:

``` js
const names = Object.keys(obj)
for (var i = 0; i < names.length; i++) {
  const name = names[i]
  this.setHeader(name, obj[name])
}
```

then we save a function allocation

File: index.js (hunk 1 above)
Comment (lamhieu-vk) [reply index.js]: we can shorthand by `return this.getAllHeaders().get(name)`, in function you missing `return`

File: index.js (hunk 2 above)
Comment (lamhieu-vk) [reply index.js]: or use:
```js
const names = Object.keys(obj)
names.forEach(key => {
  this.setHeader(key, obj[key])
})
```

File: index.js [hunk 3]
```diff
@@ -66,7 +66,8 @@ class Response {
   }
 
   setHeaders (obj) {
-    Object.keys(obj).map(key => {
+    const names = Object.keys(obj)
+    names.forEach(key => {
       this.setHeader(key, obj[key])
     })
```
Comment (lamhieu-vk) [inline index.js]: @mafintosh please check agin 👍

Comment (mafintosh) [reply index.js]: Ah sorry. This is fine, but there is a conflict with index.js.

**Pull request closed without merging**